packages/core/src/components/autosuggest/autosuggest.variables.scss

```diff
@@ -1,5 +1,13 @@
 @use "~dso-toolkit/src/variables/colors";
+@use "~dso-toolkit/src/variables/units";
 
 $background-color: colors.$wit;
 $box-shadow: 0 6px 12px rgba(0, 0, 0, 0.175);
 $border-color: rgba(0, 0, 0, 0.15);
+
+$listbox-padding-block: units.$u1;
+$listbox-border-width: 1px;
+$list-item-padding-block: units.$u1 * 0.5;
+$list-item-block-size: 24px + 2 * $list-item-padding-block;
+$listbox-max-block-size: 10 * $list-item-block-size + 2 * $listbox-padding-block + 2 * $listbox-border-width;
+$listbox-min-block-size: $list-item-block-size + 2 * $listbox-padding-block;
```

Comment:
We zijn hier bezig met het `autosuggest`-component, maar de variabelen heten `$list-` en `$listbox`. Is dat niet enigszins verwarrend?

Reply:
Oh, ja misschien wel. De naamgeving die ik hier heb gebruikt volgt die van hetgeen in de tsx-file wordt gebezigd. 🤷 

Heb jij een voorstel of voorbeeld waar ik e.e.a. op kan baseren/verduidelijken/verbeteren?

Reply:
Ik zou sowieso beginnen met `$autosuggest-`. Daarna kan je het zelf naar bel;ieven invullen. Zo koppel je de variabelen expliciet aan het component, zodat niemand in de verleiding komt om ze elders te gaan gebruiken.

Reply:
Goed punt!

Maar op zich hoeven we niet meer te namespacen, daar hebben we Sass modules voor:

```scss
// my-component.variables.scss
$size: 10px;

// my-component.scss
@use "my-component.variables" as my-component-variables;

.element {
  inline-size: my-component-variables.$size;
}
```

Bijvoorbeeld Alert: https://github.com/dso-toolkit/dso-toolkit/blob/master/packages/dso-toolkit/src/components/alert/alert.variables.scss

Niet alle variables doen dit al. Dit is voornamelijk legacy, we maken nog niet zo lang gebruik van Sass modules en waren vroeger belast met de vreselijke taak om deze variables zelf te namespacen.

Reply:
@tfrijsewijk nailed it.

Kennelijk heel lastig om dat 'oude' principe uit m'n systeem te krijgen 😀

Reply:
Ik lees dit nog eens na, maar in dit geval zou `listbox` natuurlijk een intern stukje kunnen zijn. Er is niets mis met het benoemen van onderdelen van een component en dat in TSX/markup/styling gebruiken :)

Reply:
Lijkt mij prima